client/components/clonePost/index.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-underscore-dangle */
+/* global React, crNonce */
+
+const {
+  data: {
+    select,
+  },
+  i18n: {
+    __,
+  },
+} = wp;
+
+/**
+ * Slotfill to add clone & replace controls to the sidebar.
+ */
+const ClonePost = () => {
+  const currentPost = select('core/editor').getCurrentPost();
+
+  /**
+   * We only are interested in draft posts here.
```

Comment:
The comment and the code don't seem to match - is this what is intended?
```suggestion
   * We aren't interested in draft posts here.
```